packages/falter-berlin-dynamic-banner/files/refresh_banner.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Get dynamic information and construct a new banner. This script
+
+HOSTNAME=$(uci get system.@system[0].hostname)".olsr"
+IPADDR=$(uci get network.wireless0.ipaddr)
```

Comment:
```uci get network.wireless0.ipaddr``` does not work on a device without wireless, for example the er-x-sfp or rb750gr3.  Maybe it would be better to get this information from somewhere else.   ```uci get network.dhcp.ipaddr``` should work wherever the wizard has been used, but also many "backbone" images would need to use ```uci get network.lan.ipaddr```

Reply:
I've a question for `uci get network.dhcp.ipaddr`. With this I will get another ip address than the first mesh-ip. I think that both (mesh-ip and first ip of dhcp) will work to reach the node. Should we maybe discuss for a convention which one of these addresses to use preferably?

Reply:
Yes, it's another address than the first mesh ip.  I'm totally fine with using the dhcp address.  It's also the address of frei.funk.

Another option would be to list all the IP's with labels.  For example, going through the freifunk zone and running "ubus call network.interface.X status" to get the ip addresses (both v4 and v6).  That way, any nodes which have meshlan or wireless bridge set up will have multiple addresses listed.

But I must honestly say that from all the info presented, the IP address is the least interesting to me.

Reply:
> But I must honestly say that from all the info presented, the IP address is the least interesting to me.

I agree. We could drop it. I included it, because @kls0e suggested it in issue #59.
If we decide to drop it not, I will just use the dhcp-address, as you suggested.

Reply:
solved in f8c3f58b2414f1c4a17eaea8301f7b00a1d3768c